### DISCUSSION TITLE
ForwardMessage怎么取MiraiCode

### DISCUSSION THREAD

**Discussion by Nambers** [mamoe/mirai#1096] [Q&A]
ForwardMessage可以用MiraiCode形式发送吗?如果可以那怎么取到MiraiCode
```
a.build().sendTo(c)
logger.info("B ${a.build().toMessageChain().serializeToMiraiCode()}")
```
我试过了这种，第一行可以发出去，不过第二行没有显示
![image](https://user-images.githubusercontent.com/35139537/111035036-4cc40980-8453-11eb-9151-60a2ae16ac0f.png)

**Comment by Karlatemp** [comment]
ForwardMessage未支持mirai码表现形式，只有 [表中](https://github.com/mamoe/mirai/blob/dev/docs/Messages.md#%E7%94%B1-codablemessage-%E5%8F%96%E5%BE%97-mirai-%E7%A0%81%E5%AD%97%E7%AC%A6%E4%B8%B2) 列出的才支持mirai码

**Reply by Nambers** [reply]
> ForwardMessage未支持mirai码表现形式，只有 [表中](https://github.com/mamoe/mirai/blob/dev/docs/Messages.md#%E7%94%B1-codablemessage-%E5%8F%96%E5%BE%97-mirai-%E7%A0%81%E5%AD%97%E7%AC%A6%E4%B8%B2) 列出的才支持mirai码

行吧 不过我记得MessageSource里好像是可以有转发信息应该我记错了，会考虑支持吗?
